modules/charon-core/src/main/java/org/wso2/charon3/core/extensions/UserManager.java

```diff
@@ -46,6 +46,13 @@ public User getUser(String id, Map<String, Boolean> requiredAttributes)
     public void deleteUser(String userId)
             throws NotFoundException, CharonException, NotImplementedException, BadRequestException;
 
+    default List<Object> listUsersWithGET(Node node, Integer startIndex, Integer count, String sortBy, String sortOrder,
```

Comment:
Seems like this line exceeds the max line length.

Reply:
It seems like here it's exceeding. But in the editor it does not exceed.